grafana/values.yaml

```diff
@@ -30,6 +30,8 @@ datasources: # @schema description: List of datasources to configure for the Gra
   tempo: # @schema description: Tempo datasource configuration.
     enabled: true # @schema description: Enable the Tempo datasource.
     url: "http://tempo.loki.svc:3100" # @schema description: URL of the Tempo datasource.
+  cloudwatch: # @schema description: CloudWatch datasource configuration.
+    enabled: true # @schema description: Enable the CloudWatch datasource.
```

Comment:
@alexlokshin-czi should i make this false by default?

Reply:
true won't hurt us.